lms/envs/common.py

```diff
@@ -3239,4 +3239,4 @@
 COURSEGRAPH_JOB_QUEUE = LOW_PRIORITY_QUEUE
 
 ############## Settings User Statistics ############################
-EDEOS_API_STATISTICS_URL = ''
+EDEOS_API_URL = ''
```

Comment:
Is it really needed?

Reply:
if it is not set, it complicates the local env installation. A dev might not need this config and might omit it in the json file and in this case there will be an error. So we set it to empty string in this base file